install/helm/gloo/crds/enterprise.gloo.solo.io_v1_AuthConfig.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
 spec:
   group: enterprise.gloo.solo.io
   names:
+    categories:
```

Comment:
As part of this work, I think we ran into some questions about how CRDs are generated. https://github.com/solo-io/solo-kit/tree/main/pkg/code-generator/schemagen#implementation has some of these details, though I imagine a brief README, that links to the solo-kit implementation, could be useful so that future devs know how to udate our CRDS. what do you think?

Reply:
As part of this, I see that the CRDs have `# Code generated by solo-kit. DO NOT EDIT.` and perhaps we should clarify that is somewhat true.

I think it's created because the files are edited by solo-kit, and it uses https://github.com/solo-io/solo-kit/blob/main/pkg/code-generator/schemagen/crd.go#L39. We could update this to adjust the header to be more accurate

Reply:
I added a README. Imo adjusting the header should probably be a separate task, as this one doesn't touch solo-kit in the end, and I'm not sure how easy that is to do for just CRDs and not other files. Wdyt?

Reply:
Yeah I like it